content/pages/privacy-policy.md

```diff
@@ -0,0 +1,93 @@
+
+The 3rd party that can be contacted by Mixxx is:
+
+ - MusicBrainz - For this service, please refer their Privacy Policy here: [https://metabrainz.org/privacy](https://metabrainz.org/privacy)
+
+### Broadcasting
+
+If you use the broadcasting feature of the Mixxx application, you need to enter login data of a third party Soundcast/Icecast server.
+Mixxx itself doesn't offer such a service. Only metadata of the played tracks are send to these servers by the Mixxx application.
+
+### Mixxx Crash Reporting
+
+The Mixxx application itself neither collects crash reports nor shares them.
+
+Crash reports are handled by the Stores/Vendors of each platforms.
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx Core Developers have access to these reports, which usually contain:
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+In addition Mixxx writes a mixxx.log file which contains entries that may expose personal data. In a support case we may ask to share this file in public. This is a manual process during which you may obfuscate sensitive date before sharing.  
+
+### Mixxx Update Mechanism
+
+The Mixxx application itself doesn't provide an update mechanism.
+
+In case updates are handled by the platform stores the user has downloaded Mixxx from,
+please refer to those platforms for information about the privacy policies.
+
+## 2. Main Mixxx Website
+
+The main website of Mixxx, aka www.mixxx.org, does not create or use any user account.
+There are no registration forms, or any similar mechanism.
+Therefore, no personal information is stored at any moment, about the user, by Mixxx.
+
+## 3. Mixxx Community Websites
+
+The Mixxx community websites are related to development and community, and are the following:
+
+- GitHub (Code, Issue Tracking, Wiki)
+- Discourse Forum
+- Zulip Chat
+
+Each of those websites have different users system, but work in a similar way. In order to participate users need to create a login and a user profile which requires login data and cookies. Refer the privacy statements of these sites for details:
+
+- You can find the GitHub Privacy Statement here: [https://docs.github.com/de/site-policy/privacy-policies/github-privacy-statement](https://docs.github.com/de/site-policy/privacy-policies/github-privacy-statement)
+- You can find the Privacy Policy of the Mixxx Discourse Forum here: [https://mixxx.discourse.group/privacy](https://mixxx.discourse.group/privacy)
+- You can find the Zulip Privacy Policy here: [https://zulip.com/policies/privacy](https://zulip.com/policies/privacy)
+
+One can modify their user information at any time, through the options of those websites.
+
+## Contact Mixxx
+
+Various contact options are listed here: [Contact Mixxx](/contact)
+
+*Last updated: 27 June 2023*

```

Comment:
We have the issue here that the policy is pointless without a legal entity. I am afraid we need a real person here.

Reply:
Creating a legal entity is out of scope of this PR. But I think we have to offer a solution to contact us, without creating an additional acoount at Discourse or Zulip.
What about a mail address privacy@mixxx.org ?

Reply:
According GDPR https://gdpr-info.eu/art-13-gdpr/ we need a person in control of the collected data, when we collect them. I think the only data we collect ourself, are the trafic logs of mixxx.org. Therefore the easiest way might be disabling any logging.

Reply:
Our copy right holder entity is "Mixxx Development Team" maybe we can reuse this term here?

Reply:
As far as I have read, the privacy responsible does not have to be a natural person. Therefore `Mixxx Development Team` should be fine.

Reply:
Do you have a source for this info?
Since this is all just for the MS Store, shold we Mixxx dev team, apply for the store and wait for the review?

Reply:
I think Microsoft does not check the policy itself. They just require an URL of the policy - and I added none.
We failed check of 10.5.1 of the Microsoft Store policy: https://learn.microsoft.com/en-us/windows/apps/publish/store-policies#105-personal-information.

Reply:
> Do you have a source for this info?

In German: https://www.haufe.de/compliance/recht-politik/datenschutz-grundverordnung/dsgvo-datenschutzbeauftragter-oder-verantwortlicher_230132_454140.html